scripts/spot-grid.py

```diff
@@ -53,12 +58,22 @@ def new_image(self, data):
         at ratio 1:1)
         data (numpy.ndarray): an 2D array containing the image (can be 3D if in RGB)
         """
-        self.app.spots, trans, scale, rot, shear = FindGridSpots(data, self.gridsize)
-        self.app.translation = trans[0], data.shape[0] - trans[1]
-        self.app.scale = scale
-        self.app.rotation = -rot
-        self.app.shear = shear
-
+        tform_ji = estimate_grid_orientation_from_img(
+            data,
+            (8, 8),
+            AffineTransform,
+            sigma=1.45,
+            threshold_rel=0.5,
+        )
+        grid = unit_gridpoints((8, 8), mode="ji")
+        self.app.spots = tform_ji.apply(grid)
+
+        tform_xy = to_physical_space(tform_ji, data.shape, pxsize)
```

Comment:
`to_physical_space` needs to be imported and pxsize is not defined here, can be solved by setting self.pxsize in the init

Reply:
I think there is also still a bug in the `to_physical_space` for a transform. I checked this code and `tform_xy.scale` is exactly equal to `tform_ji.scale`